source/extensions/filters/http/rbac/rbac_filter.cc

```diff
@@ -65,13 +65,30 @@ RoleBasedAccessControlRouteSpecificFilterConfig::RoleBasedAccessControlRouteSpec
 
 Http::FilterHeadersStatus RoleBasedAccessControlFilter::decodeHeaders(Http::HeaderMap& headers,
                                                                       bool) {
+  ENVOY_LOG(
+      debug,
+      "checking request:\n"
```

Comment:
Debug lines in general should be single line. Please remove the newlines.

Reply:
Done